src/linear_algebra/basic.lean

```diff
@@ -300,6 +300,13 @@ instance {T : Type*} [monoid T] [distrib_mul_action T M₂] [smul_comm_class R T
   smul_comm_class S T (M →ₗ[R] M₂) :=
 ⟨λ a b f, ext $ λ x, smul_comm _ _ _⟩
 
+-- example application of this instance: if S -> T -> R are homomorphisms of commutative rings and
+-- M and M₂ are R-modules then the S-module and T-module structures on Hom_R(M,M₂) are compatible.
```

Comment:
A vague guess at docgen-ifying this:
```suggestion
/--
Example application of this instance: if S -> T -> R are homomorphisms of commutative rings and
`M` and `M₂` are `R`-modules then the `S`-module and `T`-module structures on `M →ₗ[R] M₂` are compatible.
-/
```

Reply:
Yes exactly -- it means we have a commutative triangle of ring homomorphisms S->T, T->R and S->R. It's how mathematicians use the X -> Y -> Z idiom, which is completely different to how it is used here. The docstring is fine -- thanks.

Reply:
Were you planning to accept this change and forgot, or decided it didn't matter?